docs/gambit/index.md

```diff
@@ -21,14 +21,15 @@ We built [Gambit](https://github.com/Certora/gambit),
 We use the mutants generated by Gambit to assess the CVL specifications
   written for a verification task.
 Gambit is a general-purpose mutation generation tool that can be used with many
-kinds of verifiers or testing tools.  {doc}`gambit` describes how Gambit can be
-used to generate mutants for use with any tool; while {doc}`mutation-verifier`
-describes how to use Gambit with the Certora Prover to evaluate specifications.
+  kinds of verifiers or testing tools. 
+{doc}`mutation-verifier` describes how to use Gambit with the Certora Prover to 
+  evaluate specifications, 
+  while {doc}`gambit` describes how Gambit can be used to generate mutants for use with any tool.
```

Comment:
I would change the order
gambit - generic tool for generating mutants
mutation-varifier - tool that integrates gambit to cetora prover

Reply:
We wanted this order so that users would see the most relevant thing first